team-roles/readme.md

```diff
@@ -0,0 +1,15 @@
+# Team roles
```

Comment:
I would consider moving the team-roles section into team-agreements. What do you think?

Reply:
File was removed.